spec/controller_services/shopping_list_items_controller/destroy_service_spec.rb

```diff
@@ -40,17 +40,15 @@
           expect(perform.resource).to be nil
         end
 
-        it 'sets the updated_at timestamp on the shopping list', :aggregate_failures do
+        it 'sets the updated_at timestamp on the shopping list', do
           t = Time.now + 3.days
-
           Timecop.freeze(t) do
             perform
             # use `be_within` even though the time will be set to the time Timecop
             # has frozen because Rails (Postgres?) sets the last three digits of
             # the timestamp to 0, which was breaking stuff in CI (but somehow not
             # in dev).
-            expect(shopping_list.reload.updated_at).to be_within(0.05.seconds).of(t)
-            expect(aggregate_list.reload.updated_at).not_to be_within(0.05.seconds).of(t)
```

Comment:
The aggregate list should have its timestamp changed too but it isn't important because when the order of shopping lists matters the aggregate list is first regardless.